code/obj/item/gun/kinetic.dm

```diff
@@ -2094,6 +2094,40 @@ ABSTRACT_TYPE(/obj/item/survival_rifle_barrel)
 		set_current_projectile(new/datum/projectile/bullet/rod)
 		..()
 
+/obj/item/gun/kinetic/four_bore_albatross
+	name = "\improper Albatross four-bore rifle"
+	desc = "A behemoth of a scoped rifle developed by Cormorant Precision Arms. Intended for suppression or elimination of monstrous targets."
+	icon = 'icons/obj/items/guns/kinetic64x32.dmi'
+	wear_image_icon = 'icons/mob/clothing/back.dmi'
+	icon_state = "four_bore"
+	item_state = "four_bore"
+	w_class = W_CLASS_BULKY
+	flags =  FPRINT | TABLEPASS | CONDUCT | USEDELAY | EXTRADELAY
+	c_flags = EQUIPPED_WHILE_HELD | ONBACK
+	slowdown = 10
+	slowdown_time = 8
+	force = MELEE_DMG_RIFLE
+	two_handed = TRUE
+	can_dual_wield = FALSE
+	contraband = 7
+	ammo_cats = list(AMMO_FOUR_BORE)
+	spread_angle = 2
```

Comment:
bit odd to have spread on a scoped weapon imo

Reply:
I was basing this off old elephant guns, and thought it was funny to include their inaccuracy past short range despite having scopes- to note, this scope only goes out to 16 tiles. I can remove the inaccuracy, if you want.